packages/aws-cdk/lib/api/hotswap/cloudformation-executable-template.ts

```diff
@@ -0,0 +1,230 @@
+import * as cxapi from '@aws-cdk/cx-api';
+import * as AWS from 'aws-sdk';
+import { ListStackResources } from './common';
+
+export interface CloudFormationExecutableTemplateProps {
+  readonly stackArtifact: cxapi.CloudFormationStackArtifact;
+  readonly parameters: { [parameterName: string]: string };
+  readonly account: string;
+  readonly region: string;
+  readonly partition: string;
+  readonly urlSuffix: string;
+
+  readonly listStackResources: ListStackResources;
+}
+
+export class CloudFormationExecutableTemplate {
```

Comment:
💅 Why *executable* ? Why not something like `EvaluateCloudFormationTemplate`? Or if you must have a noun: `CloudFormationTemplateEvaluator` ?

Reply:
Sure, changed to `EvaluateCloudFormationTemplate`.